src/main/java/org/carlmontrobotics/subsystems/Limelight.java

```diff
@@ -89,16 +90,15 @@ public double getDistanceToNoteMeters() {
     }
   }
 
-  public double getArmAngleToShootSpeakerRad(){
+  public double getArmAngleToShootSpeakerRad() {
     double armRestingHeightToSubwooferMeters = HEIGHT_FROM_RESTING_ARM_TO_SPEAKER_METERS;
     double horizontalDistanceMeters = getDistanceToSpeakerMeters() + SIDEWAYS_OFFSET_TO_OUTTAKE_MOUTH;
     return END_EFFECTOR_BASE_ANGLE_RADS - Math.atan(armRestingHeightToSubwooferMeters / horizontalDistanceMeters);
   }
 
-  public double getRotateAngleDeg() {
+  public double getRotateAngleRad() {
     double cameraLensHorizontalOffset = getTXDeg(SHOOTER_LL_NAME) / getDistanceToSpeakerMeters();
     double realHorizontalOffset = Math.atan(cameraLensHorizontalOffset / getDistanceToSpeakerMeters());
```

Comment:
Don't think it matters in this case but wanted to bring up that there exists a [`Math.atan2`](https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#atan2-double-double-) which can return angles within the domain [pi, -pi] instead of [-pi/2, pi/2].
Also do you account for the case when the apriltag is not detected?

Reply:
if the apriltag is not detected, `getRotateAngleRad()` will return 0 radians. `getRotateAngleRad()` is only used in AlignToApriltag.java. this causes the robot to turn 0 radians if the command is run, so nothing happens.

Reply:
Why would we need to be able to return an angle over 90deg? Our camera does _not_, in fact, have 360deg FOV.

Reply:
I was mentioning that atan2 exists for future reference cuz that method is a little obscure. I wasn't trying to get you to actually use that method. Sorry for the confusion. Resolved